### ISSUE TITLE
`P` to create a function in a shellcode blob no longer works

### ISSUE THREAD

**Issue by psifertex** [Vector35/binaryninja-api#1971]
Looks like a regression since last stable, pressing `P` is mapped to both:
`Create Function Here`
and
`Make Function at This Address – Default Platform`
and neither can fire on latest dev:
```
Action 'Make Function at This Address\Default Platform' has an ambiguous shortcut
Action 'Create Function' has an ambiguous shortcut
```

**Comment by toolCHAINZ** [comment]
FYI I get this when manually making functions in ELFs and stuff too.

**Comment by jeffli678** [comment]
fixed in dev 2.1.2397
